### ISSUE TITLE
Marble: split package marble desktop and marble maps, ship marble maps in KCI

### ISSUE THREAD

**Issue by notmart** [blue-systems/pm#192]
package marble maps, from the marble repository, master branch.
at the moment marble maps doesn't have a desktop file, so may have to be delayed, but they promised one.
also, the package should be splitted to have marble desktop and marble maps in different packages

**Comment by aleixpol** [comment]
Having heard the conversation, I think it would be useful to provide them a Review Request with a patch that adds marble-mobile.desktop file...

**Comment by jriddell** [comment]
I've updated the packaging and it's building now in kci mobile

**Comment by jriddell** [comment]
built now on armhf, waiting on publisher to stop being busy to publish

**Comment by jriddell** [comment]
built

**Comment by star-buck** [comment]
reopen - not closing, but mark "Ready for QA" until confirmed fixed on an actual IMG.

**Comment by star-buck** [comment]
not seeing any marble in v4 IMG...

**Comment by apachelogger** [comment]
@jriddell maybe not installed by mobster?

**Comment by jriddell** [comment]
coming in today's new image

**Comment by jriddell** [comment]
on image 20150924.130033

**Comment by star-buck** [comment]
marble maps icon on homescreen, but app doesnt start

**Comment by star-buck** [comment]
closing this now and open new ticket
